src/main/clojure/clara/rules.cljc

```diff
@@ -332,6 +332,12 @@
       * :forms-per-eval - The maximum number of expressions that will be evaluated per call to eval.
         Larger batch sizes should see better performance compared to smaller batch sizes. (Only applicable to Clojure)
         Defaults to 5000, see clara.rules.compiler/forms-per-eval-default for more information.
+      * :omit-compile-ctx - When false Clara, in Clojure, retains additional information to improve error messages during
+        session deserialization at the cost of additional memory use.
```

Comment:
I don't know if it has to be specifically stated for deserialization.
It retains compile context that can be useful for certain types of inspection. Deserialization error message reporting is one example.

Reply:
Perhaps just say "eg. during deserialization" or as part of the doc, not making it sound like it is exclusively a durability flag (of course that's the only use _we_ have for it in the compiler right now).